### ISSUE TITLE
Modernize Docs Theme

### ISSUE THREAD

**Issue by Mierdin** [StackStorm/st2docs#565]
The current theme is showing its age. In particular, literally 40% of the page is totally blank, unused space due to the fact that the content body is so narrow:

![screen shot 2017-07-28 at 10 43 13 am](https://user-images.githubusercontent.com/4230395/28730244-1c64855a-7384-11e7-8251-6563c5c512b5.png)

As you can see, this presents problems for content that must occupy a certain amount of space.

We should consider adopting a more modern-looking theme, one that uses the full screen. Some notable examples are:

- [Kubernetes](https://kubernetes.io/docs/concepts/overview/kubernetes-api/)
- [Salt](https://docs.saltstack.com/en/latest/ref/configuration/minion.html)

**Comment by warrenvw** [comment]
+1

**Comment by LindsayHill** [comment]
Yeah, @enykeev and I were chatting about it a while back. There's some bug fixes we could do - e.g. bring across the latest change to the Sphinx RTD theme we use - but it is time for a proper overhaul.

We're going to need to re-work the theme for bwc-docs.brocade.com. Might be time to address both then?
